### ISSUE TITLE
Create RollBack utility

### ISSUE THREAD

**Issue by rtisma** [overture-stack/SONG#327]
create python script to interact with score, song and s3, to clean up accidentally uploaded stuff.

should have python library with the functions to execute the cleanup. Once that is done, can add CLI which will pass in a TSV file with a specific schema.

depends on #326

**Comment by andricDu** [comment]
out of scope.
